in-depth/submission_guidelines.md

```diff
@@ -3,31 +3,31 @@ layout: docs
 title: Theme & Plugin Submission Guidelines
 headline: Theme & Plugin Submission Guidelines
 description: |
-  Submitting your Theme or Plugin for inclusion on the website is very easy.<br>
+  Submitting your theme or plugin for inclusion on the website is very easy.<br />
```

Comment:
Can I ask why you always choose to "close" tags like this?

As far as my understanding goes: In a modern, HTML5 based document, this isn't necessary.

It was only briefly a thing when XHTML was proposed to be the replacement for HTML 4.01. (For xml-related reasons)

And as far as compatibility reasons go, there shouldn't be any browser that would render `<br />`, but not `<br>`.

Once upon a time, I also had trained myself to do this, but I've since broken that habit.

So, I've just been wondering what your reasoning is for this? :thinking: